packages/mint-components/src/components/tax-and-cash/sqm-user-info-form/small-views/DocusignWrapperView.tsx

```diff
@@ -0,0 +1,115 @@
+import { VNode, h } from "@stencil/core";
+import { createStyleSheet } from "../../../../styling/JSS";
+
+export interface DocusignWrapperSlotViewProps {
+  states: {
+    loading: boolean;
+    formState: {
+      docusignSlot: VNode;
+      completedTaxForm: boolean;
+      errors?: any;
+    };
+  };
+  callbacks: {
+    onChange: (e) => void;
+  };
+  text: {};
+}
+
+const style = {
+  Container: {
+    width: "100%",
+    display: "flex",
+    justifyContent: "flex-start",
+    flexDirection: "column",
+    gap: "16px",
+  },
+  RadioContainer: {
+    width: "100%",
+    display: "flex",
+    justifyContent: "flex-start",
+    flexDirection: "column",
+    gap: "16px",
+    padding: "16px",
+  },
+  DescriptionText: {
+    color: "var(--sl-color-neutral-500)",
+    lineHeight: "22px",
+  },
+  BoldText: {
+    fontWeight: "bold",
+  },
+
+  Link: {
+    color: "#14B1F7",
+    textDecoration: "none",
+    "&:visited": {
+      color: "#14B1F7",
+    },
```

Comment:
We should consider some color tokens for links since this comes up relatively often.

Reply:
File has been deleted